components/form_inputs/radio_buttons/VariousAttributesRadioButtons.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-container fluid>
+   <v-row align="center">
+     <span class="mr-3">Success, Hint, Background-color, Readonly</span>
```

Comment:
Please add ID for each group of radio buttons

Reply:
DONE